### PR TITLE
Tweak data markers

### PR DESCRIPTION
**Pull request by gitjul**
Before:

Screenshot 2019-05-09 at 00 18 25

After:

Screenshot 2019-05-09 at 00 18 01

### CODE REVIEW

File: app/assets/stylesheets/modules/markers.css.scss [hunk 1]
```diff
@@ -30,47 +31,47 @@
...
+
   &.mid {
-    border: 1px solid #ffcc33;
+    border: 1px solid $yellow;
   }
+
   &.low {
     border: 1px solid $green;
   }
-  &.lowest {
```
Comment (gitjul) [inline app/assets/stylesheets/modules/markers.css.scss]: lowest is mapped to the `default` class in `_heat.js`

**Comment by gitjul** [comment]
this is pure styles but feel free to review @AkkarinForest @3v0k4

**Comment by 3v0k4** [comment]
👁

**Comment by gitjul** [comment]
thx @3v0k4!

**Pull request merged**